doc/man3/OSSL_CRMF_MSG_get0_tmpl.pod

```diff
@@ -40,6 +43,11 @@ OSSL_CRMF_MSG_get_certReqId
                                         EVP_PKEY *pkey);
 
  int OSSL_CRMF_MSG_get_certReqId(const OSSL_CRMF_MSG *crm);
+ OSSL_CRMF_CERTTEMPLATE *OSSL_CRMF_CERTTEMPLATE_dup(const OSSL_CRMF_CERTTEMPLATE *a);
+
+ OSSL_CRMF_ATTRIBUTETYPEANDVALUE 
+ *OSSL_CRMF_ATTRIBUTETYPEANDVALUE_dup(const OSSL_CRMF_ATTRIBUTETYPEANDVALUE *a);
+ void OSSL_CRMF_ATTRIBUTETYPEANDVALUE_free(OSSL_CRMF_ATTRIBUTETYPEANDVALUE *a);
```

Comment:
Missing doc text for these functions.
Better move them to the generic `X509_dup.pod`.